ngrinder-frontend/src/js/components/perftest/list/List.vue

```diff
@@ -420,7 +420,7 @@
                         }
 
                         this.tests[target.index].status = updatedStatus;
-                        this.runningSummary = `${res.data.perfTestInfo.length} ${this.i18n('perfTest.list.runningSummary')}`;
+                        this.runningSummary = `${res.data.runningTestsCount} ${this.i18n('perfTest.list.runningSummary')}`;
```

Comment:
It unnecessarily exposed other users' performance test information. So, changed it.